cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -34,9 +34,11 @@
 
 
 class TLM:
```

Comment:
remaining suggestions:

prompt_async and the get_trustworthiness_score_async 

still need more explanation why somebody would want to use it in their top-level docstrings